packages/api-reference/src/stores/globalStore.ts

```diff
@@ -16,6 +16,10 @@ export const createEmptyAuthenticationState = (): AuthenticationState => ({
   apiKey: {
     token: '',
   },
+  oAuth2: {
```

Comment:
Shouldn't this be owned be the ApiClient store?

Reply:
That’s a really good question! I thought of the auth stuff in the reference as an extra layer that we should keep in a different store. But I guess it would make it simpler to just have one store.

Currently, if I edit the credentials in the API client, it has no impact on the credentials in the reference. If we keep only one store, editing credentials in the API client would also change what’s in the reference. 

Is that what we want? 🤔 Give me some time to think about that.

Reply:
> If we keep only one store, editing credentials in the API client would also change what’s in the reference.

@marclave @cameronrohani thoughts?

Reply:
I think we want a global store, cause when we feed in the auth from outside of the references or even from intro, it should be fed into the client. however, i think when you are editing in the client its kind of fair to think of it as an isolated sandbox environment with seeded information.